src/tools/kata-ctl/src/version.rs.in

```diff
@@ -14,3 +14,20 @@ const KATA_CTL_VERSION: &str = "@KATA_CTL_VERSION@";
 pub fn get() -> String {
     format!("{}-{}", KATA_CTL_VERSION, crate_version!())
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use semver::{Version};
```

Comment:
Are there guidelines for putting `use` statements here vs at the top of a file? I'm not suggesting you move them, genuinely wondering.

Reply:
yes, it is because the semver module is just called in the tests scope, this also helps to prevent warning messages when semver is declared globally.